e2e/profile.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { test, expect, type Page } from '@playwright/test';
```

Comment:
```suggestion
import { test, expect } from '@playwright/test';
```

Remove the import because...